### ISSUE TITLE
Possible NU3 Goal - Payments to Sprout (Zcash 1.0) zaddrs are consensus-invalid

### ISSUE THREAD

**Issue by mms710** [zcash/zcash#3771]
Possible NU3 goal - implement changes to protocol that disallows payments to Sprout. Here are some related issues: https://github.com/zcash/zcash/issues/3677 (could be a duplicate if we don't do this as a minor release/non-consensus-change) and https://github.com/zcash/zcash/issues/3752 (maybe helpful to do as well)

**Comment by daira** [comment]
I immediately went to close this as a duplicate of #3677 when I saw the title :-) When there are multiple ways to do a thing, we don't usually open separate tickets for them.

**Comment by mms710** [comment]
Oh excellent, thanks for catching that @daira !! I'll close this as a duplicate of https://github.com/zcash/zcash/issues/3677 and make a note that #3677 is dependent on #3752.
